client/app/components/projects/applicant-project-card.hbs

```diff
@@ -5,6 +5,15 @@
 >
   <h3 class="no-margin">
     {{@project.dcpProjectname}}
+
+    {{#if (eq @project.statuscode "On-Hold")}}
+      <span class="text-red text-weight-normal"
+            data-test-project-on-hold
+      > 
+        (On Hold) 
+      </span>
+    {{/if}}
```

Comment:
This is great.

Could even have some new computeds on the project model:

`{{#if (eq @project.isOnHold)}}`

Love it.

Reply:
@allthesignals I originally created the project card components, when the markup between the two was more different.  They have since become basically the same, and totally on the same page -- one of the small refactors I was hoping to get to before the end of this sprint is to create one project card component that's reusable across the lists.

Reply:
Also interested to chat about the idea of moving that to a computed -- I'm not sure I understand Ember data models yet to know what benefits that'd get us!

Reply:
Hmm yeah we can talk. Computed properties can be made on anything, including ember data models, because models themselves extend the object model.

It would get us closer to the convention of keeping business logic out of templates, although it's so minor here it doesn't matter. I have, however, seen it get pretty out of hand, though — lots of nested conditionals in templates.